fontbe/src/orchestration.rs

```diff
@@ -219,8 +220,99 @@ impl From<&VariationRegion> for TupleBuilder {
     }
 }
 
+#[repr(u8)]
+#[derive(Copy, Clone, Debug, PartialEq)]
+pub enum LocaFormat {
+    Short,
+    Long,
+}
```

Comment:
could be explicit about the values here if you like:
```suggestion
    Short = 0,
    Long = 1,
}
```

and then you can avoid matching later and just do `format as u8 as _`. You could also do `repr(u16)` and then just cast directly.

Reply:
(to clarify: this is sort of redundant since this is how enums work in general, but since we do actually care about these values I think there's some value in being explicit)